### PR TITLE
Early review: Remove NetworkObject and NetworkService

### PR DESCRIPTION
**Pull request by eriknordmark**
Still need a NetworkObject in the parser since parseSystemAdapter uses it. (That will go away with the new model). Renamed that to NetworkXObject.

### CODE REVIEW

**Comment by rvs** [comment]
LGTM

File: pkg/pillar/cmd/zedagent/parseconfig.go [hunk 1]
```diff
@@ -1331,51 +1204,40 @@ func parseUnderlayNetworkConfigEntry(
...
-			return nil
-		}
-		ulCfg.UsesNetworkInstance = true
-		networkUuidStr = networkInstanceEntry.Uuidandversion.Uuid
-		log.Infof("NetworkInstance(%s-%s): InstType %v\n",
+	// Lookup NetworkInstance ID
+	networkInstanceEntry := lookupNetworkInstanceId(intfEnt.NetworkId,
+		cfgNetworkInstances)
+	if networkInstanceEntry == nil {
+		ulCfg.Error = fmt.Sprintf("App %s-%s: Can't find network id %s in networks "+
```
Comment (gkodali-zededa) [inline pkg/pillar/cmd/zedagent/parseconfig.go]: Log message should exclude the "network id" part.

File: pkg/pillar/cmd/zedagent/parseconfig.go [hunk 2]
```diff
@@ -1446,35 +1308,28 @@ func parseOverlayNetworkConfigEntry(
...
-	} else {
-		if !isOverlayNetworkObject(netEnt) {
-			return nil
-		}
-		olCfg.UsesNetworkInstance = false
+	// Lookup NetworkInstance ID
+	networkInstanceEntry := lookupNetworkInstanceId(intfEnt.NetworkId,
+		cfgNetworkInstances)
+	if networkInstanceEntry == nil {
+		olCfg.Error = fmt.Sprintf("App %s - Can't find network id %s in networks or "+
```
Comment (gkodali-zededa) [inline pkg/pillar/cmd/zedagent/parseconfig.go]: log message should exclude "network id" part

File: pkg/pillar/cmd/zedagent/parseconfig.go (hunk 1 above)
Comment (eriknordmark) [reply pkg/pillar/cmd/zedagent/parseconfig.go]: Fixed

File: pkg/pillar/cmd/zedagent/parseconfig.go (hunk 2 above)
Comment (eriknordmark) [reply pkg/pillar/cmd/zedagent/parseconfig.go]: Fixed

**Pull request merged**